src/main/java/software/amazon/awssdk/crt/CrtResource.java

```diff
@@ -36,11 +37,11 @@ public abstract class CrtResource implements AutoCloseable {
     /**
      * Debug/diagnostic data about a CrtResource object
      */
-    public class ResourceInstance {
+    public static class ResourceInstance {
```

Comment:
just curious, why? I don't understand why an inner class would be static vs non-static